src/entities/swap/swaps/v3/index.ts

```diff
@@ -97,30 +99,44 @@ export class SwapV3 implements SwapBase {
 
         return this.isBatchSwap
             ? this.queryBatchSwap(client, block)
```

Comment:
Would likely need to do similar for batchSwap.

Reply:
Could use help understanding best methods for debugging errors with SDK. Like should I be using debugger with breakpoint and "Step into" ?

Reply:
I'd just go with whatever you normally use. I use a combo of console logs to narrow down the issue and debugger if i need to see a bunch of variables.